test/unit/Context/PhpDocumentorContextFactoryTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Roave\BetterReflectionTest\Context;
+
+use phpDocumentor\Reflection\Types\ContextFactory;
+use Roave\BetterReflection\Context\PhpDocumentorContextFactory;
+use phpDocumentor\Reflection\Types\Context;
+
+class PhpDocumentorContextFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateForNamespace()
+    {
+        $namespace = 'Foobar';
+        $source = <<<EOT
+<?php 
+
+namespace Foobar;
+
+use Barfoo as Foo;
+use Foobar as Boo;
+EOT
+        ;
+
+        $innerContextFactory = new ContextFactory();
+
+        $contextFactory = new PhpDocumentorContextFactory($innerContextFactory);
```

Comment:
Remove inner context factory. This object constructs the "php documentor" context factory itself, it has no arguments.